config/alienwhitelist.txt

```diff
@@ -22,6 +22,7 @@ flaktual - Vox
 flurriee - Protean
 funnyman2003 - Xenochimera
 hawkerthegreat - Vox
+H0lySquirr3l - Protean
```

Comment:
I recommend lowercase.

Reply:
But thats my name username as I type it in?

Reply:
Code handles things a little differently.